metabase/metabase
# fix audit cards have actions menu

Closes https://github.com/metabase/metabase/issues/31570

### Description

Removes dashcard menus from the audit dashboards.

### How to verify

Run Metabase EE

1. Open Admin -> Audit
2. A special audit dashboard appears with no dashcard borders
3. Hover any dashcard, ensure there is no menu in the right top corner of a dashcard

### Checklist
